tests/tasks.py

```diff
@@ -293,3 +292,40 @@ def mytask(longer_arg):
             eq_(arg.names, ('longer-arg', 'l'))
             eq_(arg.attr_name, 'longer_arg')
             eq_(arg.name, 'longer_arg')
+
+        def decorated_function_names_are_preserved(self):
```

Comment:
I was able to get tests to run fine with `invoke test`, but I wasn't able to run individual tests or individual test files. Everytime I tried to use `nosetest` or `spec` on `tests/tasks.py`, it wouldn't detect any tests. So, how do you run individual tests?